app/templates/base-flat.coco.pug

```diff
@@ -41,116 +41,117 @@
   block footer
     #footer.small
       if !me.hideFooter()
-        .container
-          .row
-            if me.showChinaResourceInfo()
-              .col-lg-12
-                .row
-                  .col-lg-3
-                    h3(data-i18n="nav.general")
-                    ul.list-unstyled
-                      li
-                        a(href="/about", data-i18n="nav.about")
-                      //li
-                      //  a(href="/about#careers", data-i18n="nav.careers")
-                      if !me.isStudent()
+        if !getQueryVariable('landing', false)
```

Comment:
would have preferred condition to be added to hideFooter line itself to avoid indentation changes 😅

Reply:
That's where I started, but I still needed parts of it.